github.go

```diff
@@ -2,24 +2,28 @@ package resource
 
 import (
 	"errors"
+	"io"
 	"net/url"
 	"os"
 
 	"code.google.com/p/goauth2/oauth"
 
-	"github.com/google/go-github/github"
+	"github.com/zachgersh/go-github/github"
 )
 
 //go:generate counterfeiter . GitHub
 
 type GitHub interface {
 	ListReleases() ([]github.RepositoryRelease, error)
+	LatestRelease() (*github.RepositoryRelease, error)
+	GetReleaseByTag(tag string) (*github.RepositoryRelease, error)
 	CreateRelease(release *github.RepositoryRelease) (*github.RepositoryRelease, error)
 	UpdateRelease(release *github.RepositoryRelease) (*github.RepositoryRelease, error)
 
-	ListReleaseAssets(release *github.RepositoryRelease) ([]github.ReleaseAsset, error)
+	ListReleaseAssets(release github.RepositoryRelease) ([]github.ReleaseAsset, error)
 	UploadReleaseAsset(release *github.RepositoryRelease, name string, file *os.File) error
 	DeleteReleaseAsset(asset github.ReleaseAsset) error
+	DownloadReleaseAsset(asset *github.ReleaseAsset) (io.ReadCloser, error)
```

Comment:
Could probably make all of these non-pointers in the future; the github API returns `*Foo` in the singular case and `[]Foo` in the many case, so they're probably more intended to be passed around by-value (and they're only a pointer in the singular case to avoid returning an empty one). Which granted is a little odd since every single field in them is a pointer, making them not really by-value, but whatever.

Not at all blocker for this PR (it was already inconsistent, imo the google github api is actually inconsistent), but it'd make the bit down below a little less odd (having `.downloadAsset` take a pointer).